modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala

```diff
@@ -17,7 +17,7 @@ import scala.reflect.ClassTag
 import scala.util.{ Failure, Success, Try }
 
 object Java {
-  implicit class RichType(val tpe: Type) extends AnyVal {
+  implicit class RichType(private val tpe: Type) extends AnyVal {
```

Comment:
Out of curiosity: why these changes?

Reply:
#### `scalafmt: PreferCurlyFors` (1547e1d)
Rewrites `for (a <- fa; b <- fb) yield (a, b)` (bafflingly, the syntax scalafix prefers) into the long-form
```scala
for {
  a <- fa
  b <- fb
} yield (a, b)
```

#### `scalafix: RemoveUnused` (2fe53d9)
Stripped out some unused imports. I still think there are more, I'll try running this again a few more times.

#### `scalafix: ExplicitResultTypes` (86f2554)
There were a few public functions that didn't have an explicit result type, which this rule fixed.

#### `scalafix: LeakingImplicitClassVal` (31d1162)
This one was somewhat surprising:

```scala
@ implicit class RichString(val foo: String) { def bang = s"$foo!" }
defined class RichString

@ "blep".bang
res3: String = "blep!"

@ "blep".foo
res4: String = "blep"
```

Resolved if you don't have the `val`,
```scala
@ implicit class RichString(foo: String) { def bang = s"$foo!" }
defined class RichString

@ "blep".bang
res1: String = "blep!"

@ "blep".foo
cmd2.sc:1: value foo is not a member of String
val res2 = "blep".foo
                  ^
Compilation Failed
```

but more completely resolved by just marking the field private.